src/components/common/Notifications/index.tsx

```diff
@@ -2,21 +2,43 @@ import { ReactElement, SyntheticEvent, useCallback, useEffect } from 'react'
 import groupBy from 'lodash/groupBy'
 import { useAppDispatch, useAppSelector } from '@/store'
 import { closeNotification, Notification, selectNotifications } from '@/store/notificationsSlice'
-import { Alert, AlertColor, Snackbar, SnackbarCloseReason } from '@mui/material'
+import { Alert, AlertColor, Link, Snackbar, SnackbarCloseReason } from '@mui/material'
 import css from './styles.module.css'
+import NextLink from 'next/link'
+import { useRouter } from 'next/router'
+import ChevronRightIcon from '@mui/icons-material/ChevronRight'
 
 const toastStyle = { position: 'static', margin: 1 }
 
-const Toast = ({ message, severity, onClose }: { message: string; severity: AlertColor; onClose: () => void }) => {
+const Toast = ({
+  message,
+  variant,
+  link,
+  onClose,
+}: {
+  message: string
+  variant: AlertColor
+  link?: Notification['link']
+  onClose: () => void
+}) => {
+  const router = useRouter()
+
   const handleClose = (_: Event | SyntheticEvent, reason?: SnackbarCloseReason) => {
     if (reason === 'clickaway') return
     onClose()
   }
 
   return (
-    <Snackbar open onClose={handleClose} sx={toastStyle} autoHideDuration={severity === 'success' ? 5000 : null}>
-      <Alert severity={severity} onClose={handleClose} elevation={3} sx={{ width: '340px' }}>
+    <Snackbar open onClose={handleClose} sx={toastStyle} autoHideDuration={variant === 'success' ? 5000 : null}>
```

Comment:
I added this `autoHideDuration` a while back and @katspaugh mentioned that with the notification center we could auto hide all notifications by default, not just the success ones. Wdyt?

Reply:
It makes sense for them to automatically hide, but `safe-react` includes the possibility of persisting notifications. I'll make them all autohide for now and we can revisit it again at a later stage.